storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -12,12 +12,11 @@ toc::[]
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the AWS EBS CSI Driver Operator. The Operator provides a default StorageClass that you can use to create PVCs. You also have the option to create the EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
+To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-After the Operator is installed, you must also create the AWS EBS CSI custom resource (CR) that is required in the {product-title} cluster.
+* The **AWS EBS CSI Driver Operator** provides a default StorageClass that you can use to create PVCs. You also have the option to create the AWS EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
 
-:FeatureName: AWS EBS CSI Driver Operator
-include::modules/technology-preview.adoc[leveloffset=+1]
+* The **AWS EBS CSI driver** is a custom resource (CR) that enables you to create and mount AWS EBS PVs.
 
```

Comment:
"AWS EBS CSI driver" is not a CR. IIUC, it includes some resources like deployment, daemonset..., the resources work together to enable the service of creating/mounting PVs.

Reply:
@bobfuru The CRD ManilaDriver and AWSEBSDriver do not exist in OCP4.6.  They are the CRs needed when the operators are managed by OLM.

Reply:
Thank you, @qinpingli - that makes sense now that I realize the CRs are OLM-based. :)